data/yam/autoyast/support_images/sled15sp5_install_all_patterns_x86_64.xml

```diff
@@ -0,0 +1,203 @@
+    # quote : "during the post-script phase, YaST still has an exclusive lock on the RPM database."
+    # the package cannot be installed the autoyast way, because of the changing package name (eg libyui-rest-api15)
+    # only zypper allows to install "by capability".
+    mv /var/run/zypp.pid /var/run/zypp.sav
+    zypper -n in libyui-rest-api
+    mv /var/run/zypp.sav /var/run/zypp.pid
+
+exit 0
+
+]]></source>
+      </script>
+    </post-scripts>
+  </scripts>
+  <services-manager t="map">
+    <default_target>graphical</default_target>
+    <services t="map">
+      <disable t="list"/>
+      <enable t="list">
+        <service>sshd</service>
+      </enable>
+    </services>
+  </services-manager>
+  <software t="map">
+    <packages t="list">
+      <package>sle-module-desktop-applications-release</package>
+      <package>sle-module-python3-release</package>
+      <package>sle-we-release</package>
+      <package>sle-module-basesystem-release</package>
+    </packages>
+    <patterns t="list">
+      <pattern>base</pattern>
+      <pattern>minimal_base</pattern>
+      <pattern>enhanced_base</pattern>
+      <pattern>documentation</pattern>
+      <pattern>apparmor</pattern>
+      <pattern>x11</pattern>
+      <pattern>x11_enhanced</pattern>
+      <pattern>yast2_basis</pattern>
+      <pattern>sw_management</pattern>
+      <pattern>fonts</pattern>
+      <pattern>32bit</pattern>
+      <pattern>gnome</pattern>
+      <pattern>gnome_x11</pattern>
+      <pattern>gnome_multimedia</pattern>
+      <pattern>gnome_imaging</pattern>
+      <pattern>office</pattern>
+      <pattern>technical_writing</pattern>
+      <pattern>books</pattern>
+      <pattern>gnome_basic</pattern>
+      <pattern>multimedia</pattern>
+      <pattern>laptop</pattern>
+      <pattern>imaging</pattern>
+    </patterns>
+    <products t="list">
+      <product>SLED</product>
+    </products>
+  </software>
+  <suse_register t="map">
+    <addons t="list">
+      <addon t="map">
+        <arch>x86_64</arch>
+        <name>sle-module-desktop-applications</name>
+        <version>15.5</version>
+      </addon>
+      <addon t="map">
+        <arch>x86_64</arch>
+        <name>sle-module-python3</name>
+        <version>15.5</version>
+      </addon>
+      <addon t="map">
+        <arch>x86_64</arch>
+        <name>sle-we</name>
+        <reg_code>{{SCC_REGCODE_WE}}</reg_code>
+        <version>15.5</version>
+      </addon>
+      <addon t="map">
+        <arch>x86_64</arch>
+        <name>sle-module-basesystem</name>
+        <version>15.5</version>
+      </addon>
+    </addons>
+    <do_registration t="boolean">true</do_registration>
+    <install_updates t="boolean">true</install_updates>
+    <reg_code>{{SCC_REGCODE}}</reg_code>
+  </suse_register>
+  <users t="list">
+    <user t="map">
+      <encrypted t="boolean">false</encrypted>
+      <fullname>Bernhard M. Wiedemann</fullname>
+      <user_password>nots3cr3t</user_password>
```

Comment:
Minor comment, you can use encrypted passed here

Reply:
ok, I tried to use encrypted passed, but it can not login after installation, I'll try again.

Reply:
I updated it, and running VRs, thanks.